node/src/components/block_synchronizer/execution_results_acquisition.rs

```diff
@@ -18,16 +22,16 @@ use crate::types::{
 #[derive(Clone, Copy, PartialEq, Eq, DataSize, Debug, Serialize, Deserialize)]
 pub(crate) enum ExecutionResultsChecksum {
     // due to historical reasons, pre-1.5 chunks do not support Merkle proof checking
-    Uncheckable,
+    ApprovalsUncheckable,
     // can be Merkle proof checked
-    Checkable(Digest),
+    ApprovalsCheckable(Digest),
```

Comment:
It's a bit confusing to name the enum variants `Approvals…`; the fact that approvals and execution results are checkable for the same blocks is kind of accidental, and not relevant in most places where this type is used.